source/about/self-hosted-subscriptions.rst

```diff
@@ -58,7 +70,20 @@ Subscription and licenses not purchased via the Customer Portal won't be listed.
 Add more users to your subscription
 -----------------------------------
 
-To add more users to your existing subscription, `contact us <https://mattermost.com/contact-us/>`__.
+If you have a self-hosted license in the Customer Portal, you can add additional users to your subscription via the `Customer Portal page <https://customers.mattermost.com>`__.
+
+1. Select **Purchase additional seats**.
+2. Enter your account and billing details.
+3. When the transaction is complete, select whether you'd like to download your updated license or having it emailed to you.completes, they will have to either
+
+Once you have your updated license, upload it via **System Console > License and Edition**.
+
+This process adds additional users to your existing subscription and is not a new license. Your license renewal date doesn't change when you add additional users and receive an updated license. Billing is pro-rated based on the time left in your billing cycle.
+
+If you are using Mattermost v7.6 and below, to add more users to your existing subscription, `contact us <https://mattermost.com/contact-us/>`__.
```

Comment:
Missed this one @justinegeffen -- I guess what would be more appropriate here is that if they don't have a self-hosted license in the Customer Portal and wanted to add users, contact us